src/components/helpers/pretty-text-input.js

```diff
@@ -531,10 +512,18 @@ export default createReactClass({
 
   removeCodeMirrorEditor: function() {
     const textBoxNode = this.textBoxRef;
+
     const cmNode = textBoxNode.firstChild;
     textBoxNode.removeChild(cmNode);
+
     this.codeMirror.off('change', this.onCodeMirrorChange);
     this.codeMirror.off('focus', this.onFocusCodeMirror);
+
+    if (this.debouncedOnChangeAndTagCodeMirror) {
+      this.debouncedOnChangeAndTagCodeMirror.cancel();
+      this.debouncedOnChangeAndTagCodeMirror = null;
+    }
+
```

Comment:
Make sure we cancel any trailing invocation so we are safe everywhere else.

Reply:
Uh, hmm, how do I tell you this... the `_.debounce` here is not actually lodash's debounce. It's Formatic's "undash" debounce: https://github.com/zapier/formatic/blob/48ceba41b2c04917e1ac1449dfadf77544d725bc/src/undash.js#L74-L108

It's a fake lodash with just the bare necessities. So if you want to be able to call `cancel`, you'll have to change the implementation. Which is pretty straightforward.

Reply:
Haha I got baited 😄 Thanks for pointing it out, I will implement the `cancel`.